qcodes/instrument/base.py

```diff
@@ -368,11 +368,8 @@ def call(self, func_name: str, *args: Any) -> Any:
     def __getstate__(self) -> None:
         """Prevent pickling instruments, and give a nice error message."""
         raise RuntimeError(
-            'Pickling {}. qcodes Instruments should not.'.format(self.name) +
-            ' be pickled. Likely this means you '
-            'were trying to use a local instrument (defined with '
-            'server_name=None) in a background Loop. Local instruments can '
-            'only be used in Loops with background=False.')
+            'Pickling {}. QCoDeS instruments can not'.format(self.name) +
+            ' be pickled.')
```

Comment:
just a few formatting suggestions:
```suggestion
            f'Error when pickling instrument {self.name}. '
            f'QCoDeS instruments can not be pickled.')
```

Reply:
I agree that we should use f-strings.